### ISSUE TITLE
it doesn't find the password

### ISSUE THREAD

**Issue by simonbcn** [dannyvankooten/browserpass#12]
Firefox 50
Arch Linux 64 bits

I've added the login information in pass for github:
```
$ pass github
XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
login: xxxxxxx
url: github.com
```

But `browserpass` doesn't find it:
![men _001](https://cloud.githubusercontent.com/assets/428231/20640854/2964e280-b3eb-11e6-9951-31aa1e8b9820.jpg)

I have executed the `install.sh` file:
```
$ ./install.sh                

Select your browser:
====================
1) Chrome
2) Chromium
3) Firefox
1-3: 3

Installing Firefox host config
Native messaging host has been installed to /home/juan/.mozilla/native-messaging-hosts.
```
The created file:
```
$ cat /home/juan/.mozilla/native-messaging-hosts/com.dannyvankooten.browserpass.json
{
  "name": "com.dannyvankooten.browserpass",
  "description": "Browserpass binary for the Firefox extension",
  "path": "/home/juan/bin/browserpass-linux64/browserpass-linux64",
  "type": "stdio",
  "allowed_extensions": [
     "browserpass@dannyvankooten.com"
   ]
}
```

**Comment by simonbcn** [comment]
I have imported the lastpass csv data and now it works, but why it hasn't worked before?
Now this entry is:
```
$ pass Bugs/github.com 
xxxxxxxxxxxxxxxxxxx
---
Bugs / github.com
username: xxxxxxx
password: xxxxxxxxxxxxxxxxxxx
url: https://github.com/login
```
How should I added a new entry to work?

**Comment by dannyvankooten** [comment]
Hi @simonbcn,

The extension will search for the current domain by default, where the name of the password file has to match the domain (host). In your example the password file should be named `github.com` if you want the login to show right away when opening up the extension.

Alternatively you should see the login when searching for "github".  

The extension does not yet look in the pass file for a line with the website URL and I am not sure whether I will ever add that as it means the extension will have to go through all password files & decrypt them one by one just to find a line with a matching website URL.

TL;DR: You can use the following format for creating new entries.

```
$ pass insert github.com
---------
************
username: yourusername
```

Hope that clarifies a little!
